bundles/net.tourbook.chart/src/net/tourbook/chart/ComputeChartValue.java

```diff
@@ -1,14 +1,14 @@
 /*******************************************************************************
  * Copyright (C) 2005, 2018 Wolfgang Schramm and Contributors
```

Comment:
@gmngueko 
Minor change: When you modify a file, make sure to update the date: Here it would be : 2005, 2021